clustersConfig.py

```diff
@@ -456,6 +461,7 @@ def dpu_mac_address(a: int) -> str:
         template.globals['worker_number'] = worker_number
         template.globals['worker_name'] = worker_name
         template.globals['api_network'] = api_network
```

Comment:
My understanding is that the api network is the same as the iso network api port?

Reply:
no, it's not.

CDA contains `network_api_port: {{ api_network() }}` when deploying the OCP cluster (That is, this port on the provisioning host is connect to an interface on the worker nodes.

For cluster-kind `iso`, `network_api_port: {{ iso_network_api_port() }}` is an interface on the provisioning host which is in the same VLAN as an interface on the DPUs.

We just agreed that I will rename `iso-network-api-port` to `secondary`. I will do that.
Afterwards, I will also update the spread-sheet to change the "yes" in the "Provision Host" column to `primary`.

Reply:
yes, we are currently using a separate network for the host and another for the dpu (+ the workloads on the host). In the future, we will add primary networking support and having that naming in place today will avoid confusion going forward. Thanks